src/tests/enforce-boolean-naming-prefixes.test.ts

```
@@ -0,0 +1,305 @@
+  ],
+  invalid: [
+    // Variables without proper boolean prefixes
+    {
+      code: 'const active = true;',
+      errors: [
+        {
+          messageId: 'missingBooleanPrefix',
+          data: {
+            type: 'variable',
+            name: 'active',
+            prefixes: 'is, has, does, can, should, will, was, had, did, would, must, allows, supports, needs',
```

ESLint

[error] Replace `prefixes:` with `··prefixes:⏎···············`

(prettier/prettier)